WordPress/Sniffs/PHP/EnqueuedCheckSniff.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPress\Sniffs\PHP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Based off the StrictInArraySniff.php Sniff, THis checks the Enqueued 4th Parameter to make sure a Version is available.
+ * The Enqueued functions are:
+ * wp_register_script()
```

Comment:
Use:

```php
* - `wp_register_script()`
```

for each item here, so that extracted docs will display it as a list, with code markup.